### PR TITLE
Add Endpoint to delete Group

### PR DESCRIPTION
**Pull request by pmanrubia**
## DO NOT MERGE BEFORE #97
Only commit: 1d3baca781cee9685a33f3713653360460992c03 applies

### Description

* Add endpoint to delete a group

### Sample

```terminal
curl "http://content-performance-manager.dev.gov.uk/groups.json?api_token=1234567" \
     -H "Content-Type: application/json"
```

### CODE REVIEW

File: doc/api.md [hunk 1]
```diff
@@ -49,3 +49,10 @@ curl "http://content-performance-manager.dev.gov.uk/groups/3.json?api_token=1234
 curl "http://content-performance-manager.dev.gov.uk/groups.json?api_token=1234567" \
      -H "Content-Type: application/json"
 ```
+
+* Delete a group
+
+```terminal
+curl "http://content-performance-manager.dev.gov.uk/groups.json?api_token=1234567" \
```
Comment (chao-xian) [inline doc/api.md]: I think this example is missing a param to specify which group to delete.

Comment (pmanrubia) [inline doc/api.md]: Thank you for the review.
I have addressed your comment in 70cac7e

**Comment by pmanrubia** [comment]
@chao-xian I have addressed your comments, please let me know if you are happy with the PR

File: doc/api.md (hunk 1 above)
Comment (chao-xian) [inline doc/api.md]: That worked but now I have a question about ID vs slug as param which I've asked in https://github.com/alphagov/content-performance-manager/pull/96/commits/6ac84c3c1338bb4236abcf310389304b2c946b82#r107136016

**Pull request merged**